src/app/services/cryptoWrapper.ts

```diff
@@ -1,5 +1,7 @@
 import { AES, SHA256, WordArray, enc } from 'crypto-js';
+import { Injectable } from "@angular/core";
```

Comment:
@ronaldmansveld - minor one: suggest switching to single quotes for consistency.